packages/api/src/external/carequality/organization.ts

```diff
@@ -23,6 +23,10 @@ export type CQOrgDetails = {
 
 export async function createOrUpdateCQOrganization(): Promise<void> {
   const cqOrgDetailsString = Config.getCQOrgDetails();
+  if (!cq) {
+    const msg = "No CQ API key found. Skipping...";
+    throw new Error(msg);
+  }
```

Comment:
nit: could prob be a one liner 😄 

For the message, we're tying the empty cq to the the API KEY - this code doesn't have info about this, that's one of the benefits of using the factory (on the top of the file). Maybe just say something like `CQ API is not enabled` or something?